snyk/models.py

```diff
@@ -662,9 +662,14 @@ def _aggregated_issue_to_vulnerabily(
         ).all()
 
         try:
-            upgrade_path = next(
-                filter(lambda path: path[0].fixVersion is not None, issue_paths.paths)
-            ).map(format_package)
+            upgrade_path = list(
+                map(
+                    format_package,
+                    filter(
+                        lambda path: path[0].fixVersion is not None, issue_paths.paths,
```

Comment:
is there any test that can be updated for this?

Reply:
There weren't any applicable tests so I added one that hits this path